app/controllers/api/v2/versions_controller.rb

```diff
@@ -3,6 +3,8 @@ class Api::V2::VersionsController < Api::BaseController
 
   def show
     return unless stale?(@rubygem)
+    cache_expiry_headers
+    set_surrogate_key "gem/#{@rubygem.name}"
```

Comment:
is this going to share a surrogate key with the v1 versions api responses?

Reply:
yes, which is intended. this is the key that gem cache purger purges

Reply:
so surrogate key is only about purging
it doesnt imply content equivalence
only a key that can be used for purging
so in this case, we want them to be the same